src/task2/server_interface.py

```diff
@@ -0,0 +1,32 @@
+import connection
+import json
+
+def to_json(obj):
+    return json.dumps(obj, separators=(",", ":"))
+
+class serverInterface:
+    def __init__(self, name):
+        self.opened_connection = connection.Connector()
+        self.opened_connection.connect()
+        self.opened_connection.send(connection.Action.LOGIN, to_json({"name": name}))
+        msg = self.opened_connection.receive()
+            
+    def closeConnection(self):
+        if self.opened_connection :
+            self.opened_connection.send(connection.Action.LOGOUT)
+            msg = self.opened_connection.receive()        
+            self.opened_connection.close()
+            self.opened_connection=None
+            return msg
+    
+    def getMapLevel(self,level):
+        if level == 0 or level == 1 or level == 10:
```

Comment:
here better do `if level in [0, 1, 10]` or `if level in set([0, 1, 10])`

Reply:
was fixed